docs/how-to-use-the-playground.md

```diff
@@ -121,3 +121,56 @@ FROM "metalake_demo.catalog_postgres".hr.employees AS e,
 WHERE e.employee_id = p.employee_id AND p.employee_id = s.employee_id
 GROUP BY e.employee_id,  given_name, family_name;
 ```
+
+### Lakehouse practice
```

Comment:
What is "Lakehouse practice"?

Reply:
Is it better to change to "Using Iceberg REST catalog service"?

Reply:
OK,  We don't use REST catalog service when we read data. I gave it a more abstract name.

Reply:
Fixed.